API.md

```diff
@@ -1687,6 +1687,17 @@ image.onload = function () {
   var imageTexture = regl.texture(image)
 }
 
+/* From an ImageBitmap
+ * This is only useful when loading many images, as bitmaps can be created asynchronously
+ * and will transfer faster than an image element.
+ */
+var image = new Image()
+image.src = 'http://mydomain.com/myimage.png'
+image.onload = function () {
+  createImageBitmap(image).then(function (bitmap) {
+    var imageTexture = regl.texture(bitmap)
+  }
+}
```

Comment:
One thing that's nice about `createImageBitmap` is it's availability in workers, where the `Image` constructor is not available since it's a dom object.  I hacked my local copy of regl to include this fix and use it as so:

```
const res = await fetch('http://mydomain.com/myimage.png')
const blob = new Blob(await res.arrayBuffer())
const bitmap = await createImageBitmap(blob)

// transfer bitmap to UI thread w/ worker.postMessage 
// since it's a transferable it doesn't _copy_ the image bitmap

regl.texture(bitmap)
```
Handy to load a bunch of images in workers, decode them on the GPU and add them as textures.